pnpm-workspace.yaml

```diff
@@ -1,3 +1,4 @@
 packages:
   - 'apps/*'
   - 'packages/*'
+  - 'examples/*'
```

Comment:
i'm torn if we need an examples workspace or not, what are the negatives if we just put this under apps?

Reply:
didn't want to get it confused with apps that we're making for prod use - i would have to subname everything in there as example-paymaster-nft-mint for instance.

but honestly it has almost zero switching costs if we wanna move apps out into the future. down to stick with everything in apps for now

Reply:
cool! updated